tests/flow/test_aggregation.py

```
@@ -314,3 +314,91 @@ def test_aggregate_over_map_property(self):
+            self.graph.query("MATCH (n:GW) RETURN count(*) AS c, n.v AS k ORDER BY k").result_set,
+            [[1, 1.5], [1, 9007199254740992], [1, 9007199254740993]])
+        self.env.assertEqual(
+            self.graph.query(
+                "MATCH (n:GW) WHERE n.v > 2 RETURN n.v + 0 AS k, count(*) ORDER BY k").result_set,
+            [[9007199254740992, 1], [9007199254740993, 1]])
+
+        # An error raised while evaluating a key still fails the query.
+        try:
+            self.graph.query("MATCH (p:GK) RETURN p.id / 0 AS k, count(*)")
+            self.env.assertTrue(False)
+        except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)